Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -359,14 +378,19 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
     Real const clightsq = 1.0_rt / ( PhysConst::c * PhysConst::c );
 
     // Loop over particles and deposit into Jx_arr, Jy_arr and Jz_arr
+    amrex::Real* cost_real = (amrex::Real*) amrex::The_Managed_Arena()->alloc(sizeof(amrex::Real));
+    *cost_real = 0.;
     amrex::ParallelFor(
         np_to_depose,
         [=] AMREX_GPU_DEVICE (long const ip) {
-
+#if (defined AMREX_USE_GPU)
```

Comment:
If we implement a proper fallback in `KernelTimer`, then we should not need this extra `#ifdef` guard here, which would make the whole thing even more compact in the code sections where we inject it :)

Reply:
Sounds good, that will make this cleaner.  I'm happy to do this in a followup, I'll put it on my to-do list.

Reply:
The function
```
_ZN5amrex13launch_globalIZNS_11ParallelForIlZ27doEsirkepovDepositionShapeNILi3EEvRK19GetParticlePositionPKdS7_S7_S7_PKiRKNS_6Array4IdEESD_SD_ldRKSt5arrayIdLm3EESF_NS_4Dim3EdiPdlEUllE_vEENSt9enable_ifIXsr5amrex19MaybeDeviceRunnableIT0_vEE5valueEvE4typeERKNS_3Gpu10KernelInfoET_OSM_EUlvE_EEvST_
```
uses 4 more registers when compiled with the `KnlTimer`:
```
No KnlTimer
(Esirkepov) ptxas info    : Used 72 registers, 736 bytes cmem[0], 24 bytes cmem[2]
Yes KnlTimer
(Esirkepov) ptxas info    : Used 76 registers, 760 bytes cmem[0], 24 bytes cmem[2]
```